shared/android/app/build.gradle

```diff
@@ -6,7 +6,7 @@ apply from: '../../node_modules/react-native-unimodules/gradle.groovy'
 import com.android.build.OutputFile
 
 // KB: app version
-def VERSION_NAME = "4.0.0"
```

Comment:
missed this last time?

Reply:
i think it was a function of the build changing to app bundling